package.json

```diff
@@ -75,30 +60,26 @@
     "jsdoc": "3.4.0",
     "jshint": "2.9.1",
     "json-loader": "0.5.4",
-    "load-grunt-config": "0.17.2",
-    "load-grunt-tasks": "3.2.0",
     "memdown": "1.0.0",
+    "mocha": "2.4.5",
```

Comment:
@cdaringe @rsskmr 
Yeah, see this branch in ansible-ops where the npm installs happen: https://github.com/MRN-Code/ansible-ops/pull/24/files 
roles/nodejsserver/tasks/npm.yml <---

I think @rsskmr could fill us in there? (I didn't make that specific change)